ci/scripts/build.sh

```diff
@@ -4,8 +4,8 @@ cwd=$(pwd)
 
 export GOPATH=$cwd/go
```

Comment:
This variable won't be used anymore, so it is probably best to remove it for cleanliness.